build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -64,6 +61,18 @@
     # (torch_models.EFFICIENTNET_B7_FP32_TORCH, [8]),
 ]
 
+x86_64_MODELS_AND_THREADS_LONG = [
```

Comment:
By default we are building all benchmarks regardless if they are long-running benchmarks. This makes the total size of the benchmark suite exceed 200GB (and increases the build time to 17mins). We need to split the build target first.

Reply:
We spoke offline about this. Are you planning to add changes to split up the compilation? If so, I'll hold off on submitting this PR.

Reply:
Yes, I'm working on it https://github.com/openxla/iree/pull/13129

Reply:
https://github.com/openxla/iree/pull/13665 is merged. This should be unblocked now.